Repositories/ProjectOverview.php

```diff
@@ -143,4 +155,39 @@ public function getSelectedMilestoneColor(string $milestoneId)
 
         return $values;
     }
+
+    /**
+     * Get all projects from the database
+     *
+     * @access public
+     * @return array<string, mixed> Returns an array of all projects
+     */
+    public function getAllProjects(): array
+    {
+        $sql = 'SELECT * FROM zp_projects';
```

Comment:
this makes me nervous

Reply:
https://github.com/Leantime/leantime/blob/0e7ddbbe3d582f657a1dddfef7b3419ae588cbf7/app/Domain/Projects/Repositories/Projects.php#L61 perhaps limit with state?

Reply:
I understand. I added a where state != "1" and a LIMIT 200, just to be safe.